Update spotify.md (#181)
music-assistant/music-assistant.io: docs/music-providers/spotify.md

- Due to restrictions with Spotify's API, only Spotify Premium accounts are supported (including Duo and Family). Free accounts will not work
- When using a personal ClientID, automatically generated playlists such as "Discover Weekly" and radio mode will no longer work

!!! note 

>     If you want to have access to official "by Spotify" playlists, do not use a ClientID for authentification. Otherwise, MA will only have access to playlists created by you and other users.

Reviewer: ```suggestion
    Official "by Spotify" playlists are not available when using a personal ClientID for authentification.
```

We don't don't write the docs in second person so you can simplify this as suggested.
